mongodbatlas/fw_provider_authentication_test.go

```diff
@@ -0,0 +1,44 @@
+package mongodbatlas
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestAccSTSAssumeRole_basic(t *testing.T) {
```

Comment:
curious if for this test case we can ensure that MONGODB_ATLAS_PUBLIC_KEY and MONGODB_ATLAS_PRIVATE_KEY are not defined so we are certain that AWS authentication if being used